eth2-ssv/network/p2p/p2p_sync.go

```diff
@@ -0,0 +1,145 @@
+package p2p
```

Comment:
What are the DOS protections in the p2p send/receive pipelines? What validation can be done to avoid any bad or malicious activity overloading the software?

Reply:
We recommend using libp2p ValidatorReject or ValidationIgnore to ensure we deal with bad messages accordingly